otb-luci/files/usr/lib/lua/luci/view/otb_overview.htm

```diff
@@ -21,16 +21,33 @@ <h2><%:Overview%></h2>
     <legend><%:Paths%></legend>
     <table class="cbi-section-table" id="paths_table">
         <tr class="cbi-section-table-titles">
+            <th class="cbi-section-table-cell">Interface</th>
             <th class="cbi-section-table-cell">State</th>
             <th class="cbi-section-table-cell">Local IP</th>
+            <th class="cbi-section-table-cell">Router IP</th>
             <th class="cbi-section-table-cell">Public IP</th>
+            <th class="cbi-section-table-cell">ASN</th>
             <th class="cbi-section-table-cell">RTT</th>
             <th class="cbi-section-table-cell">MTU</th>
             <th class="cbi-section-table-cell">Upload MAX</th>
             <th class="cbi-section-table-cell">Download MAX</th>
         </tr>
         <tr class="cbi-section-table-row">
-            <td colspan="4"><em>Collecting data...</em></td>
+            <td colspan="10"><em>Collecting data...</em></td>
+        </tr>
+    </table>
+</fieldset>
+
+<fieldset class="cbi-section">
+    <legend><%:Realtime graphs%></legend>
+    <table class="cbi-section-table" id="graphs_table">
+        <tr class="cbi-section-table-titles">
+                <th class="cbi-section-table-cell">Upload</th>
+                <th class="cbi-section-table-cell">Download</th>
+        </tr>
+        <tr class="cbi-section-table-row">
+            <td><canvas width="500" height="300px" id="mountNode_tx"></canvas></td>
```

Comment:
Is this responsive ? The whole point of changing theme is to be more responsive !

Reply:
Tested on android and the whole section is horizontally scrollable